dashboard/src/components/proposals/ProposalBlock.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div v-if="proposal && proposalLikes != null" class="flex justify-between">
+    <div class="flex flex-col gap-3 justify-between">
+      <h3 class="text-xl font-semibold">
+        <a
+          :href="redirectRouteToSameWindow(proposal.route)"
+          class="hover:text-green-500 transition-colors duration-300 cursor-pointer"
+        >
+          {{ proposal.talk_title }}
+        </a>
+      </h3>
+      <div
+        class="flex flex-col gap-3 items-start mt-1 md:flex-row md:items-center"
+      >
+        <div class="flex items-center text-xs font-semibold">
+          <span class="bg-gray-200 px-2 py-1 text-gray-600 rounded-sm mr-2">{{
+            proposal.session_type.toUpperCase()
+          }}</span>
+          <div>
+            <span
+              class="px-2 py-1 rounded-sm md:hidden"
+              :class="getStatusClass(proposal.status)"
+              >{{ proposal.status.toUpperCase() }}</span
+            >
+          </div>
+        </div>
+        <div v-if="proposal.status == `Approved`" class="flex text-sm">
```

Comment:
The organizers can keep the CFPs non-anonymous as well. In that case we should be showing the details of the speaker, irrespective of whether it is Approved or Rejected

Reply:
will update based on this.

Reply:
updated